app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
Consider using parentheses for importing multiple classes/modules to adhere to the code style guide: `from app.errors import (VaccineError, NotWearingMaskError)`.